src/components/PageHeader/PageHeader.tsx

```diff
@@ -57,14 +63,18 @@ export const PageHeader = ({
     <header className={componentClassName} {...other}>
       <div className={styles['page-header__left']}>
```

Comment:
Looking at https://www.figma.com/file/0jYcObnngcbsiBOsOPUmtR/Implementation-H1-'22?node-id=2457%3A118577 and https://www.figma.com/file/yLYL19XZI0SWhbQJYuKVso/EDS-Pilot%3ACAD?node-id=1190%3A17197 I think the left and right sections are supposed to take up 50% width each for desktop?

Reply:
And the button group would be aligned to the right

Reply:
Let's get all of the content (like your stepper) in there first and then we can adjust if needed. One of those things that is hard to scope out without everything in there. Totally hear ya that we may need a 50/50 approach there. But let's get something in there and then we can adjust if that's alright

Reply:
@jinlee93  Let's get all of the content (like your stepper) in there first and then we can adjust if needed. One of those things that is hard to scope out without everything in there. Totally hear ya that we may need a 50/50 approach there. But let's get something in there and then we can adjust if that's alright

Reply:
That sounds good to me